cmd/porter/version.go

```diff
@@ -7,15 +7,18 @@ import (
 )
 
 func buildVersionCommand(p *porter.Porter) *cobra.Command {
-	opts := version.Options{}
+	opts := version.VersionOpts{}
```

Comment:
Would you please move `VersionOpts` into `pkg/porter/version.go`. The `version` package is intended for other mixins to reuse when they implement the version command. 

That will change this line to look like to `opts := porter.VersionOpts{}` but won't otherwise change much. It's just to help with the organization of the code.